README.md

```diff
@@ -1,4 +1,4 @@
 ## Cache Mission
 
 The greeting-service requires a running JDG server. In OpenShift, you
-can create one with `oc create -f greeting-service/.openshift/service.jdg.yaml`.
```

Comment:
The `.openshift/service.jdg.yaml` file is expected to be in that directory by `nodeshift` I think. @lholmquist correct me if I am wrong in this case, but as a rule that's true. Not sure how easy that will be to change at the moment, but we need to look at cost vs. benefit :)

Reply:
There's no `.openshift` anyway :-) The `.openshiftio` is needed for Launcher, and that is still present. Also Launcher strips all the `.openshiftio` directories when creating the .zip / Git repo, so this PR adds a copy to the root dir.

Reply:
I just scanned the nodeshift code, and I could be completely wrong about it having some kind of dependency on `.openshift`. Foggy brain!

Reply:
like @Ladicek said,  the reason for the .openshiftio directory is for the launcher.  man, that is confusing